llvm/lib/Target/RISCV/RISCVInstrInfoXTHeadVPseudos.td

```diff
@@ -826,6 +829,56 @@ multiclass XVPseudoUSSegStore {
   }
 }
 
+// 7.8. Vector Load/Store Segment Instructions
+// 7.8.1. Vector Unit-Stride Falut-only-first Segment Loads
+class XVPseudoUSSegLoadFFNoMask<VReg RetClass, int EEW, bits<4> NF> :
+      Pseudo<(outs RetClass:$rd, GPR:$vl),
+             (ins RetClass:$dest, GPRMem:$rs1, AVL:$avl, ixlenimm:$sew),[]>,
```

Comment:
Is it feasible to use `GPRMemZeroOffset:$rs1` rather than `GPRMem:$rs1`?
It accepts `(rs1)` for `0(rs1)`.

See: https://reviews.llvm.org/D120017

Reply:
Yes! I agree with that.
Fixed in a new commit. Thank you for pointing it out!